src/MerchantFrontend/src/app/core/services/merchant-api.service.ts

```diff
@@ -6,14 +6,15 @@ import { Product } from '../models/product.model';
 import { Authority } from '../models/authority.model';
 import { StateService } from '../state/state.service';
 import { map, take } from 'rxjs/operators';
+import { environment } from "../../../environments/environment";
 
 @Injectable({
   providedIn: 'root'
 })
 export class MerchantApiService {
 
-  private merchantSwAccessKey = 'SWSCJTWTSBF_G77_ML-5KRPWTA';
-  private apiUrl = 'https://sw6.ovh';
+  private accessKey = environment.accessKey;
```

Comment:
the access key is no static thing. we got it from the authority. This is dummy data. We don't need it in the environment. I think we have to refactor every path were it is used.

Reply:
Login always goes against the default sales channel. Therefore for our current dev environment that would be totally fine